sdk/storage/storage-queue/src/QueueServiceClient.ts

```diff
@@ -130,7 +129,7 @@ interface ServiceListQueuesSegmentOptions extends CommonOptions {
    * specify that the queue's metadata be returned as part of the response
    * body. Possible values include: 'metadata'
    */
-  include?: ListQueuesIncludeType;
+  include?: string;
```

Comment:
Is this an intentional change?

Reply:
No. But this warrants some discussion offline. We can discuss this tomorrow.

Reply:
Based on the offline discussion, added a new property ListQueuesIncludeType to avoid breaking change. Keeping this as such

Reply:
The containing type `ServiceListQueuesSegmentOptions` is not exposed in public surface though.

Reply:
The containing type `ServiceListQueuesSegmentOptions` is not exposed in public surface though so change here is fine.